### DISCUSSION TITLE
Pagination tools

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#97] [Ideas]
This is just an opinion here : 

This pagination tools is old school feature, as a user, you will never use this tools to show up only 10 per page, or 15,  it's a little gadget but not accurate I think, today is the scrolling new trend... Let's scroll ;)
Instead, we could use InfiniteScrolling feature which will load the validator list while scrolling, which could eventually betterize the uploading time of the page. 
![Screenshot 2023-05-26 at 20 36 37](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/c8a68832-7112-4cbf-a309-e9a481c3a991)

**Comment by TJ-BitCanna** [comment]
Let's see what we can do. Might be something to be added in a later release.

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- investigate possibility to exchange pagination with InfinityScroll

**Comment by TJ-BitCanna** [comment]
Tested and approved.
